src/sql/platform/telemetry/adsTelemetryService.ts

```diff
@@ -0,0 +1,189 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAdsTelemetryService, ITelemetryInfo, ITelemetryEvent, ITelemetryConnectionInfo, ITelemetryEventMeasures, ITelemetryEventProperties } from 'sql/platform/telemetry/telemetry';
+import { ILogService } from 'vs/platform/log/common/log';
+import { ITelemetryService } from 'vs/platform/telemetry/common/telemetry';
+
+
+class TelemetryEventImpl implements ITelemetryEvent {
+	constructor(
+		private _telemetryService: ITelemetryService,
+		private _logService: ILogService,
```

Comment:
probably should be consistent within a class with '_' prefix for private variables.  I'm fine with or without, but mixing seems confusing to me.